getting-started/running-gatewayd.md

```diff
@@ -29,10 +29,20 @@ Use the following command to start a PostgreSQL database server inside a contain
 docker run --rm --name postgres-test -p 5432:5432 -e POSTGRES_PASSWORD=postgres -d postgres
 ```
 
-Test your database by running the following command:
+**If `docker0` bridge exists**
 
 ```bash
 DOCKER_HOST=$(ip addr show docker0 | grep inet | grep -v inet6 | awk -F' ' '{ print $2 }' | sed 's/\/16//g')
+```
+
+**if there is no `docker0` bridge (using Docker Desktop)**
+Use the DNS name `host.docker.internal`, which resolves to the internal IP address used by the host.
+```bash
+DOCKER_HOST="host.docker.internal"
+```
+
+Run the following command to test your database:
+```bash
```

Comment:
@sinadarbouy How about using this one-liner instead? It checks whether the `docker0` interface exists and acts accordingly. Then we can also remove the above explanations.
```bash
DOCKER_HOST=$(if ip addr show docker0 > /dev/null 2>&1; then ip addr show docker0 | grep inet | grep -v inet6 | awk -F' ' '{ print $2 }' | sed 's/\/.*//'; else echo "host.docker.internal"; fi);
```

Reply:
:+1: Good idea!